source/github-helpers/parse-backticks.tsx

```diff
@@ -5,7 +5,7 @@ const splittingRegex = /`(.*?)`/g;
 export default function parseBackticks(description: string): DocumentFragment {
 	const fragment = new DocumentFragment();
 	for (const [index, text] of description.split(splittingRegex).entries()) {
-		if (index % 2 && text.length >= 1) {
+		if (index % 2 && text.length > 0) {
```

Comment:
?

Reply:
Xo rule

Reply:
Heh didn't realize it does the same thing